swift/ql/src/queries/Security/ECB-Encryption/ECBEncryption.qhelp

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>ECB should not be used as a mode for encryption. It has dangerous weaknesses. Data is encrypted the same way every time meaning the same plaintext input will always produce the same ciphertext. This behaviour makes encrypted messages vulnerable to replay attacks.</p>
+  </overview>
+
+  <recommendation>
+    <p>Use a different cipher mode such as CBC.</p>
+  </recommendation>
+
+  <example>
+    <p>The following example shows six cases of instantiating a cipher with various encryption keys and block modes. In the 'BAD' case, the mode of encrtyption is ECB, making the encryped data vulnerable to replay attacks.  In the 'GOOD' cases, the encryption mode is CBC, pretecting the encrypted data against replay attacks.</p>
```

Comment:
Typos
```suggestion
    <p>The following example shows six cases of instantiating a cipher with various encryption keys and block modes. In the 'BAD' case, the mode of encryption is ECB, making the encrypted data vulnerable to replay attacks.  In the 'GOOD' cases, the encryption mode is CBC, which protects the encrypted data against replay attacks.</p>
```

Reply:
Thanks @mchammer01 for your comments! I've merged all suggestions.